ssh_certificate_parser/__init__.py

```diff
@@ -48,6 +48,10 @@ def take_rsa_cert(raw_pubkey, byte_array):
     return RSAPublicKey(modulus=modulus, exponent=exponent, raw=raw_pubkey)
 
 
+def take_ed25519_cert(raw_pubkey, byte_array):
```

Comment:
question: do we need to pass `byte_array` here since it's not used? Can we remove it?

Reply:
@Justintime50 yes absolutely, I missed that one. PR updated.